File: VB/DefaultDocumentSettingsExample/Form1.vb
Imports DevExpress.XtraEditors
Imports DevExpress.XtraRichEdit
Imports System.ComponentModel
Imports DevExpress.XtraBars
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraRichEdit.API.Native

Namespace DefaultDocumentSettingsExample
    Partial Public Class Form1
        Inherits RibbonForm

        Public Sub New()
            'set the default font for all RichEditControls in the application
            DevExpress.XtraRichEdit.RichEditControlCompatibility.UseThemeFonts = False
            DevExpress.XtraRichEdit.RichEditControlCompatibility.DefaultFontSize = 12
            DevExpress.XtraRichEdit.RichEditControlCompatibility.DefaultFontName = "Times New Roman"
            InitializeComponent()

            AddHandler richEditControl1.EmptyDocumentCreated, AddressOf RichEditControl1_EmptyDocumentCreated
            AddHandler richEditControl1.DocumentLoaded, AddressOf RichEditControl1_DocumentLoaded
        End Sub

        Private Sub RichEditControl1_DocumentLoaded(ByVal sender As Object, ByVal e As EventArgs)
            'set the default font for the document loaded in the main RichEditControl
            richEditControl1.Document.DefaultCharacterProperties.FontName = "Arial"
            richEditControl1.Document.DefaultCharacterProperties.FontSize = 16
        End Sub

        Private Sub RichEditControl1_EmptyDocumentCreated(ByVal sender As Object, ByVal e As EventArgs)
            'change the formatting for the newly-created document
            richEditControl1.Document.DefaultCharacterProperties.ForeColor = Color.Red
            richEditControl1.Document.DefaultParagraphProperties.Alignment = ParagraphAlignment.Center
            richEditControl1.Document.AppendText("Document created at " & Date.Now.ToLongTimeString())
        End Sub

        Private Sub barBtnShowMoreForm_ItemClick(ByVal sender As Object, ByVal e As ItemClickEventArgs) Handles barBtnShowNewForm.ItemClick
            Dim frm As New RichEditForm()
            frm.Show()
        End Sub

        Private Sub barBtnLoadDoc_ItemClick(ByVal sender As Object, ByVal e As ItemClickEventArgs) Handles barBtnLoadDoc.ItemClick
            For Each form In Application.OpenForms
                If TypeOf form Is XtraForm Then
                    Dim richEditControl As RichEditControl = (TryCast(form, XtraForm)).Controls.OfType(Of RichEditControl)().FirstOrDefault()
                    If richEditControl IsNot Nothing Then richEditControl.LoadDocument("richtext.txt")
                End If
            Next form
        End Sub

        Private Sub barBtnNewDoc_ItemClick(ByVal sender As Object, ByVal e As ItemClickEventArgs) Handles barBtnNewDoc.ItemClick
            richEditControl1.CreateNewDocument(True)
        End Sub
    End Class
End Namespace

File: CS/DefaultDocumentSettingsExample/Form1.cs
using DevExpress.XtraEditors;
using DevExpress.XtraRichEdit;
using DevExpress.XtraRichEdit.API.Native;
using System;
using System.ComponentModel;
using System.Drawing;
using System.Windows.Forms;
using DevExpress.XtraBars;
using DevExpress.XtraBars.Ribbon;
using System.Linq;

namespace DefaultDocumentSettingsExample
{
    public partial class Form1 : RibbonForm
    {

        public Form1()
        {
            //set the default font for all RichEditControls in the application
            RichEditControlCompatibility.UseThemeFonts = false;
            RichEditControlCompatibility.DefaultFontSize = 12;
            RichEditControlCompatibility.DefaultFontName = "Times New Roman";
            InitializeComponent();

            richEditControl1.EmptyDocumentCreated += RichEditControl1_EmptyDocumentCreated;
            richEditControl1.DocumentLoaded += RichEditControl1_DocumentLoaded;
        }

        private void RichEditControl1_DocumentLoaded(object sender, EventArgs e)
        {
            //set the default font for the document loaded in the main RichEditControl
            Document document = richEditControl1.Document;
            document.DefaultCharacterProperties.FontName = "Arial";
            document.DefaultCharacterProperties.FontSize = 16;
        }

        private void RichEditControl1_EmptyDocumentCreated(object sender, EventArgs e)
        {
            //change the formatting for the newly-created document
            Document document = richEditControl1.Document;
            document.DefaultCharacterProperties.ForeColor = Color.Red;
            document.DefaultParagraphProperties.Alignment = ParagraphAlignment.Center;
            document.AppendText("Document created at " + DateTime.Now.ToLongTimeString());
        }
        private void barBtnShowMoreForm_ItemClick(object sender, ItemClickEventArgs e)
        {
            RichEditForm frm = new RichEditForm();
            frm.Show();
        }

        private void barBtnLoadDoc_ItemClick(object sender, ItemClickEventArgs e)
        {
            foreach (var form in Application.OpenForms)
            {
                if (form is XtraForm)
                {
                    RichEditControl richEditControl = (form as XtraForm).Controls.OfType<RichEditControl>().FirstOrDefault();
                    if (richEditControl != null)
                        richEditControl.LoadDocument("richtext.txt");
                }
            }
        }

        private void barBtnNewDoc_ItemClick(object sender, ItemClickEventArgs e)
        {
            richEditControl1.CreateNewDocument(true);
        }
    }
}
